gnucash/report/standard-reports/transaction.scm

```diff
@@ -1168,7 +1141,7 @@ tags within description, notes or memo. ")
                                                currency
                                                (gnc-numeric-neg amount))))))
                    (vector (if (and reverse?
-                                    (member (xaccAccountGetType account) account-types-to-reverse))
```

Comment:
Alternatively we can keep this UI option and add the default "Use Global Preference" which will then pull from global?